mmcv/ops/csrc/common/cuda/voxelization_cuda_kernel.cuh

```diff
@@ -101,7 +101,7 @@ __global__ void point_to_voxelidx_kernel(const T_int* coor,
   CUDA_1D_KERNEL_LOOP(index, num_points) {
     auto coor_offset = coor + index * NDim;
     // skip invalid points
-    if (coor_offset[0] == -1) return;
+    if ((index >= num_points) || (coor_offset[0] == -1)) return;
```

Comment:
Not necessary since `CUDA_1D_KERNEL_LOOP` will keep index in range(0, num_points)

Reply:
shall we use `continue` here to avoid early return? if the number of input points is so large that multiple points shall be processed by one thread? (though practically impossible)

Reply:
Yes, continue is required.